docs/configuration.md

```diff
@@ -0,0 +1,113 @@
+
+# Barong Configuration
+## Twilio configuration
+For twilio configuration we need to set such required envs
+- `BARONG_TWILIO_ACCOUNT_SID`, which acts as a twilio username
+- `BARONG_TWILIO_SERVICE_SID`, which acts as a twilio password
+- `BARONG_TWILIO_PHONE_NUMBER`, virtual phone numbers which will give you instant access to local, national, mobile, and toll-free phone numbers
+
+We have ability to set twilio with 3 different ways
+1. ```BARONG_PHONE_VERIFICATION == "twilio_sms"```
+  If you choose phone verification as twilio sms we will use send_sms [API call](https://www.twilio.com/docs/sms/send-messages)
+  Also you can add your own template for sms using `BARONG_SMS_CONTENT_TEMPLATE`
+2. ```BARONG_PHONE_VERIFICATION == "twilio_verify"```
+  In this case we will use twilio verify [API call](https://www.twilio.com/docs/verify/api)
+  There are a lot of benefits of using this API like you can send your sms via voice
+  One verification service can be used to send multiple verification tokens, it is not necessary to create a new service each time, so you can set ```BARONG_TWILIO_SERVICE_SID``` at once
+3. ```BARONG_PHONE_VERIFICATION == "mock"```
+  Use this case only for local testing
```

Comment:
With this type of verification all the numbers will be accepted and validated as a right code for any given number